phpunit.xml.dist

```diff
@@ -15,6 +15,10 @@
         <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener"/>
     </listeners>
 
+    <php>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled" />
```

Comment:
-1 for that. We do want to handle deprecations. That's the main reason why we use the bridge.

Reply:
I added this because there are some deprecated triggered by vendors that endup with failed pipeline:

```
  1x: Method "ArrayAccess::offsetExists()" might add "bool" as a native return type declaration in the future. Do the same in implementation "PhpAmqpLib\Wire\AMQPAbstractCollection" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in CallbackMessageProviderTest::test_get_with_messages_in_queue_return_message with data set #0 from Swarrot\Tests\Broker\MessageProvider

  1x: Method "ArrayAccess::offsetGet()" might add "mixed" as a native return type declaration in the future. Do the same in implementation "PhpAmqpLib\Wire\AMQPAbstractCollection" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in CallbackMessageProviderTest::test_get_with_messages_in_queue_return_message with data set #0 from Swarrot\Tests\Broker\MessageProvider

  1x: Method "ArrayAccess::offsetSet()" might add "void" as a native return type declaration in the future. Do the same in implementation "PhpAmqpLib\Wire\AMQPAbstractCollection" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in CallbackMessageProviderTest::test_get_with_messages_in_queue_return_message with data set #0 from Swarrot\Tests\Broker\MessageProvider

  1x: Method "ArrayAccess::offsetUnset()" might add "void" as a native return type declaration in the future. Do the same in implementation "PhpAmqpLib\Wire\AMQPAbstractCollection" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in CallbackMessageProviderTest::test_get_with_messages_in_queue_return_message with data set #0 from Swarrot\Tests\Broker\MessageProvider

  1x: Method "Iterator::current()" might add "mixed" as a native return type declaration in the future. Do the same in implementation "PhpAmqpLib\Wire\AMQPAbstractCollection" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in CallbackMessageProviderTest::test_get_with_messages_in_queue_return_message with data set #0 from Swarrot\Tests\Broker\MessageProvider

  1x: Method "Iterator::key()" might add "mixed" as a native return type declaration in the future. Do the same in implementation "PhpAmqpLib\Wire\AMQPAbstractCollection" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in CallbackMessageProviderTest::test_get_with_messages_in_queue_return_message with data set #0 from Swarrot\Tests\Broker\MessageProvider

  1x: Method "Iterator::next()" might add "void" as a native return type declaration in the future. Do the same in implementation "PhpAmqpLib\Wire\AMQPAbstractCollection" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in CallbackMessageProviderTest::test_get_with_messages_in_queue_return_message with data set #0 from Swarrot\Tests\Broker\MessageProvider

  1x: Method "Iterator::rewind()" might add "void" as a native return type declaration in the future. Do the same in implementation "PhpAmqpLib\Wire\AMQPAbstractCollection" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in CallbackMessageProviderTest::test_get_with_messages_in_queue_return_message with data set #0 from Swarrot\Tests\Broker\MessageProvider

  1x: Method "Iterator::valid()" might add "bool" as a native return type declaration in the future. Do the same in implementation "PhpAmqpLib\Wire\AMQPAbstractCollection" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in CallbackMessageProviderTest::test_get_with_messages_in_queue_return_message with data set #0 from Swarrot\Tests\Broker\MessageProvider

  1x: Method "SebastianBergmann\Comparator\Comparator::accepts()" might add "bool" as a native return type declaration in the future. Do the same in child class "Prophecy\Comparator\ClosureComparator" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in CallbackMessageProviderTest::test_ack from Swarrot\Tests\Broker\MessageProvider

  1x: Method "SebastianBergmann\Comparator\ObjectComparator::accepts()" might add "bool" as a native return type declaration in the future. Do the same in child class "Prophecy\Comparator\ProphecyComparator" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in CallbackMessageProviderTest::test_ack from Swarrot\Tests\Broker\MessageProvider
```

Reply:
Thank you @odolbeau, I didn't know this trick. Done, I guess